srcs/server/src/user/user.service.ts

```diff
@@ -1,12 +1,79 @@
 import { Injectable } from '@nestjs/common';
+import { CreateUserDto } from './dto/create-user.dto';
+import { UpdateUserDto } from './dto/update-user.dto';
 import { Credentials, User } from '@prisma/client';
-import { FtRegisterUserDto, LocalRegisterUserDto } from 'src/auth/dto/registerUser.dto';
-import { PrismaService } from 'src/prisma/prisma.service';
+import { PrismaService } from '../prisma/prisma.service';
+import { ResponseUserDto } from './dto/response-user.dto';
+import { PaginationQueryDto } from '../common/dto/pagination-query.dto';
+import {
+  FtRegisterUserDto,
+  LocalRegisterUserDto,
+} from '../auth/dto/registerUser.dto';
+import {
+  NoUsersInDatabaseException,
+  UserAlreadyExistsException,
+  UserNotFoundException,
+} from './exceptions/user-exceptions';
 
 @Injectable()
 export class UserService {
   constructor(private readonly prisma: PrismaService) {}
 
+  async create(createUserDto: CreateUserDto) {
```

Comment:
Hmm, I've researched this a bit and the consensus seems to be: type inference is good because you don't have to do more typing but it can be bad if ts infers the wrong type.

This is just for consistency. I wrote a return type in all my services. The only place where I don't is on controllers because they return a function from a service so there's no need to write it twice.

Again your choice

Reply:
Ok you convinced me, for consistency, and it will be more clear reading the code if we have the return types.